### DISCUSSION TITLE
用Openwrt里Adguardhome过滤广告的，怎样使用你这个？

### DISCUSSION THREAD

**Discussion by canshen1234** [Potterli20/hosts#16] [General]
OP里面的AdguardHome是添加网址的，你是直接下载，请问怎样添加下载的hosts.txt到OP里面的Adguradhome使用？

**Comment by Potterli20** [comment]
> OP里面的AdguardHome是添加网址的，你是直接下载，请问怎样添加下载的hosts.txt到OP里面的Adguradhome使用？

可以鼠标右键看到复制链接，或者在github.com/Potterli20/file/releases
选择你需要的规则。

github.com/Potterli20/hosts的里面的链接都在github.com/Potterli20/file/releases上

注:我的规则已经高达200m的大小，规则是来自全球的
好像是12个小时更新一次

**Reply by Potterli20** [reply]
如果用adgh使用该规则，请注意内存泄露
